io_msfs_gltf.py

```diff
@@ -261,11 +263,47 @@ def load_gltf_file(gltf_file_name):
     return gltf, buffer
 
 
-def create_materials(gltf):
+def get_image_paths(gltf, texture_path: pathlib.Path, report: Callable):
+    images = []
+    if 'images' in gltf and len(gltf['images']) > 0:
+        for gltf_img in gltf['images']:
+            uri = pathlib.Path(gltf_img['uri'])
+            # blender doesn't support DDS...?
+            if uri.suffix.lower() == '.dds':
+                uri = uri.with_suffix('')
+                if not uri.suffix.lower() == '.png':
+                    uri = uri.with_suffix('.png')
+                report({'WARNING'}, f'Replaced texture "{gltf_img["uri"]}" with "{uri}", DDS not supported')
+
+            images.append(texture_path.joinpath(uri))
+    return images
+
+
+def get_texture_paths(gltf, texture_path: pathlib.Path, report: Callable):
+    images = get_image_paths(gltf, texture_path, report)
+    textures = []
+    for gltf_tex in gltf['textures']:
+        source = gltf_tex['extensions']['MSFT_texture_dds']['source']
```

Comment:
This line makes an assumption that the MSFT DDS extension is used for all the textures... that holds for default aircraft but perhaps not third party as the SDK docs say MSFS supports JPG and PNG.